<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Communicate</class>
 <widget class="QWidget" name="Communicate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>75</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="message_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>251</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="show">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>390</width>
     <height>221</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_file">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>41</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/images/加.svg</normaloff>:/icon/images/加.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="restart">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
